.gitattributes

```diff
@@ -3,6 +3,10 @@
 ###############################################################################
 * text=auto encoding=UTF-8
 
+# csc/vbc are shell scripts and should always have unix line endings
+src/Compilers/CSharp/CscCore/csc text eol=lf
+src/Compilers/VisualBasic/VbcCore/vbc text eol=lf
```

Comment:
Right. The issue is that these shell scripts are included in the toolset packages. Normally, the shell scripts in our repo are only run by cloning onto a Linux/Mac machine, and git automatically chooses LF as the line ending.

However, right now the toolset packages must be built on Windows, and so the files are cloned with CRLF and then copied into the unix packages with those line endings.

When we run them on a target unix OS, the files then blow up with bad line endings.